inc/Provider.php

```diff
@@ -78,6 +78,7 @@ public function supports_filter_user() : bool {
 	 * Fetches a list of media items that are ultimately used directly by the media managaer.
 	 *
 	 * @param array $args Raw query arguments from the POST request in the media manager.
+	 * @throws Exception Thrown if an unrecoverable error occurs.
```

Comment:
I think the description can be a bit improved. Something like this, maybe?

```suggestion
	 * @throws Exception Thrown if the provider returns too many media items.
```